src/test/scala/strawman/collection/test/Test.scala

```diff
@@ -288,6 +289,37 @@ class StrawmanTest {
     println(xs17.to(List))
   }
 
+  def sortedSets(xs: immutable.SortedSet[Int]): Unit = {
+    val xs1 = xs.map((x: Int) => x.toString)
+    val xs2: immutable.SortedSet[String] = xs1
+  }
+
+  def mapOps(xs: Map[Int, String]): Unit = {
+    val xs1 = xs.map((k, v) => (v, k))
+    val xs2: strawman.collection.Map[String, Int] = xs1
```

Comment:
This example demonstrates that calling `map` with a function that returns a tuple returns a `Map`.